src/routers/router.ts

```diff
@@ -11,26 +10,20 @@ import validateApiKey from '../middleware/validateApiKey';
 import validateAdminApiKey from '../middleware/validateAdminApiKey';
 import attachRedisClient from '../middleware/attachRedisClient';
 import analytics from '../middleware/analytics';
+import { rateLimiter } from '../middleware/rateLimiter';
 
 const router = express.Router();
 
-const FIFTEEN_MINUTES = 15 * 60 * 1000;
-const REQUESTS_PER_MS = 20;
-const createDeveloperLimiter: Express.MiddleWare = rateLimit({
-  windowMs: FIFTEEN_MINUTES,
-  max: REQUESTS_PER_MS,
-});
-
 // Google Analytics
 router.use(analytics);
 
-router.get('/words', validateApiKey, attachRedisClient, getWords);
-router.get('/words/:id', validateApiKey, validId, attachRedisClient, getWord);
-router.get('/examples', validateApiKey, attachRedisClient, getExamples);
-router.get('/examples/:id', validateApiKey, validId, attachRedisClient, getExample);
+router.get('/words', rateLimiter, validateApiKey, attachRedisClient, getWords);
```

Comment:
instead of using `raterLimiter` within each route you can add the follow line before the first `router.get`

```ts
router.us(rateLimiter)
```

ref: https://expressjs.com/en/4x/api.html#app.use